client/my-sites/plans-grid/lib/is-storage-upgradeable-for-plan.ts

```diff
@@ -17,5 +17,4 @@ export const isStorageUpgradeableForPlan = ( {
 	storageOptions.length > 1 &&
 	intervalType === 'yearly' &&
 	showUpgradeableStorage &&
-	isInSignup &&
-	flowName === 'onboarding';
+	( flowName === 'onboarding' || ! isInSignup );
```

Comment:
I wish we could get rid of `flowName` from here, along with `isInSignup` (as a general approach to start removing context from `plans-grid`). Can we maybe compile these into a property and pass into the grid? 🤔 

Something for a follow-up perhaps.

Reply:
> as a general approach to start removing context from plans-grid
Gotcha 👍 

Yeah I'm happy to consolidate these properties. For a bit more context, `isInSignup` and `flowName` are temporary and will disappear entirely once we release add-on upsells to the rest of the onboarding flows.